Proposals/xgov-171.md

```diff
@@ -0,0 +1,58 @@
+author: Elijah Andreev (@its-a-setup), Nick Grechino (@fromaline)
+discussions-to: NA
+company_name: Chainspect
+category: Tools, Analytics
+focus_area: Education, Monitoring, Analytics
+open_source: No
+amount_requested: 30000
+status: Draft
+---
+
+## Abstract
+This proposal seeks funding to facilitate research and development aimed at integrating Algorand's inner transactions into the Total Transactions Per Second (TPS) calculation and introducing the additional metric "failed transactions ratio" on <a href="https://chainspect.app/"> Chainspect</a>, the biggest TPS tracker in the market. The allocated funds will support comprehensive research on Algorand's transaction structure, the refinement of TPS-tracking scripts, and the implementation of 2 new dashboard metrics for reporting Total TPS, encompassing both standard and inner transactions and reporting the ratio of failed transactions in some timeframe.
+
+## Team
+Elijah Andreev has 5+ years of experience in product management, out of which 3 years were spent in P&G focusing on the management and development of Big Data solutions. He has been in the blockchain industry for 3+ years.
+
+Nick Grechino is a full-stack engineer with more than 5 years of experience. He has been working in the blockchain space since early 2022, focusing on the development of the front-end interface for crypto trading platforms.
+
+## Experience with Algorand
+Algorand has been integrated into the Chainspect analytics platform since its inception. Over the last six months, continuous 24/7 tracking of Algorand's metrics has been conducted, and the platform has consistently received the latest features developed by the Chainspect team. Since its establishment in August 2023, Chainspect has organically amassed over 8,000 Monthly Active Users (MAUs), demonstrating early indications of Product-Market Fit (PMF) and a growing community demand for precise and timely TPS tracking.
+
+## Present Proposal
+The proposal will consist of several steps, which will be completed in a month after the proposal is approved and finalized:
+1. Deep dive into transaction structure on Algorand. This includes the research required to understand the details of inner transactions on Algorand and how they differ from inner transactions on other chains.
+2. Update of TPS calculating scripts. This includes the update of ETL scripts and DB schema to include inner transactions.
+3. Introduction of the new metric "Failed Txn Ratio", which will show the percentage of failed transactions.
+3. Update the front-end side of the Chainspect platform with 2 new metrics Total TPS, which will include all of the transactions (inner and standard ones), and "Failed Txn Ratio", which will show the percentage of failed transactions.
+4. Marketing of the new Chainspect's features. This will include posts on X/Discord and Chainspect's blog on the nature of inner transactions and the importance of understanding the ratio of failed transactions.
+
```

Comment:
Please provide public URL of where/how the community can check proposal deliverables.